public/static/docs/command-reference/import-url.md

```diff
@@ -135,8 +135,8 @@ Follow these instructions before each example below if you actually want to try
 them on your system.
 
 Start by cloning our example repo if you don't already have it. Then move into
-the repo and checkout the
-[version](https://github.com/iterative/example-get-started/releases/tag/2-remote)
+the repo and checkout
+[the revision](https://github.com/iterative/example-get-started/releases/tag/2-remote)
```

Comment:
why not version? feels like we got too formal, too precise again

Reply:
Yes, it's just more precise.

I started a whole discussion about this in https://github.com/iterative/dvc/pull/3244#pullrequestreview-348990111 but didn't get feedback and so I decided on this criteria:

> Use "version" for data or DVC-tracked files, "revision" (and that link) in the context of pure Git/ source code, and "reference" sparingly when actually talking about Git references (to revisions/commits).

Reply:
But I'll review my changes and roll back some of these for less formality (more friendliness).

Reply:
Using less strict language now. Changed in d02ccd2a and 14d4c235.